### PR TITLE
Feat image

### PR DESCRIPTION
**Pull request by ImrozNezami**
Image added
Responsive image
Circular image

Screenshot (221)

### CODE REVIEW

File: image.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+}
+
+@media only screen and (min-width: 768px) {
+  .img-responsive {
+    height: 17rem;
+    width: 17rem;
+  }
+}
+
+@media only screen and (min-width: 992px) {
```
Comment (shashidwivedi) [inline image.css]: You can use these breakpoints for media queries. You can find it in the figma link.
image

File: image.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    width: 12rem;
+  }
+}
+
+@media only screen and (min-width: 768px) {
+  .img-responsive {
+    height: 17rem;
+    width: 17rem;
+  }
+}
```
Comment (Hamdrive) [inline image.css]: There media queries are not needed at all
Responsive images can be made by just giving max width as 100% inside a parent container

**Review by Hamdrive — CHANGES_REQUESTED** [review]
Strongly suggest to redo the responsive image styling, 
apart from that rest looks good

File: image.css (hunk 2 above)
Comment (ImrozNezami) [reply image.css]: Done

File: image.css (hunk 1 above)
Comment (ImrozNezami) [reply image.css]: Done

**Review by Hamdrive — COMMENTED** [review]
Good work, changes look perfect!
Ready to merge

**Pull request merged**